aries-site/src/components/content/ContentCard.js

```diff
@@ -12,12 +16,14 @@ export const ContentCard = forwardRef(({ topic, ...rest }, ref) => {
       align="start"
       background="background-front"
       elevation={isFocused ? 'medium' : 'none'}
+      fill
       onBlur={() => setIsFocused(false)}
       onFocus={() => setIsFocused(true)}
       onMouseOut={() => setIsFocused(false)}
       onMouseOver={() => setIsFocused(true)}
       pad="medium"
       ref={ref}
+      width={{ max: maxCardWidth }}
```

Comment:
giving a restrictive `width` with a conjunction of a `fill` prop doesn't feel right, can you describe the scenario?

Reply:
https://github.com/hpe-design/design-system/pull/579#discussion_r398262088

Reply:
Removed.